tests/common/fixtures/advanced_reboot.py

```diff
@@ -71,6 +81,13 @@ def __extractTestParam(self):
         self.readyTimeout = self.request.config.getoption("--ready_timeout")
         self.replaceFastRebootScript = self.request.config.getoption("--replace_fast_reboot_script")
 
+        # Set default reboot limit if it is not given
+        if self.rebootLimit is None:
+            if self.kvmTest:
+                self.rebootLimit = 150 # Default reboot limit for kvm
+            else:
+                self.rebootLimit = 30 # Default reboot limit for physical devices
```

Comment:
what is this reboot limit? why do we have different reboot limit for kvm v.s. physical devices?

Reply:
The reboot limit is basically the limit for data plane down time. Since kvm is expected to experience a longer data plane down time, I set different default values for kvm and physical devices.